src/acuvity/apexextend.py

```diff
@@ -340,9 +371,15 @@ def __build_scan_request(
         keywords: Optional[List[str]] = None,
         access_policy: Optional[str] = None,
         content_policy: Optional[str] = None,
+        guard_config: Optional[Union[str, Path, Dict]] = None,
     ) -> Scanrequest:
         request = Scanrequest.model_construct()
 
+        # Convert None to [] so we can safely use list methods
+        keywords = keywords or []
+        redactions = redactions or []
+        analyzers = analyzers or []
```

Comment:
just change it on the function default then?

Reply:
python :) It causes error if I do that, it complains about it being None.

Reply:
ok, makes sense ... but then you should change it back to `None` again